### ISSUE TITLE
Yandex Scheme map switches to OSM at high zoom values | Яндекс Схема переключается в OSM карту при большом значении зума

### ISSUE THREAD

**Issue by kabalin** [PastVu/pastvu#216]
Регрессия после удаления Народной карты в https://github.com/PastVu/pastvu/commit/3db635d22b3499668f8e3d098bf40475e37dd08d. В [настройке Яндекс Схемы](https://github.com/PastVu/pastvu/blob/master/public/js/module/map/map.js#L265), указана Народная карта в качестве подмены при превышении значения зума выше максимального. При отсутсвии нужной карты  переключение происходит в карту по умолчанию (Космоснимки)

Regression following removal of Yandex Public map in https://github.com/PastVu/pastvu/commit/3db635d22b3499668f8e3d098bf40475e37dd08d. It has been used as [fallback map](https://github.com/PastVu/pastvu/blob/master/public/js/module/map/map.js#L265) on high zoom values for Yandex Scheme, since Yandex Public is not available, it switches to default map (Kosmosnimki).

**Comment by kabalin** [comment]
While working on this task, please check if maximum zoom values for Yandex maps are set correctly, it looks to me that zoom up to 20 is supported (according to Yandex maps interface, did not look at API), so `limitZoom` setting is probably not needed there.

**Comment by paul-k-pastvu** [comment]
Зум Яндекса нужно проверить, а вот переключение по схеме должно работать для всех карт. 
Мы не можем запрещать пользователям зумить, т.к. иначе бывает не "разлепить" точки в "густонаселенных" местах.

**Comment by kabalin** [comment]
Конечно запрещать зум не стоит, но стоит проверить везде ли уместно переключение. Скажем в случае с Warfly, нужно ли ее переключать (к примеру на Стармен) если зум превышает 18, просто останавливать на 18 как предельный зум, или масштабировать мозайку 18 до большего значения?

**Comment by aeifn** [comment]
Можно устанавливать оверзум для карты.

**Comment by kabalin** [comment]
> Можно устанавливать оверзум для карты.

Да, я его и имею ввиду под "масштабировать мозайку 18 до большего значения"

Создал #218, есть еще несоответствия.

**Comment by kabalin** [comment]
По Яндексу - максимальное значение для схемы - 20, для спутника и гибрида - 19. Исправлено в https://github.com/PastVu/pastvu/pull/219

**Comment by paul-k-pastvu** [comment]
> > Можно устанавливать оверзум для карты.
> Да, я его и имею ввиду под "масштабировать мозайку 18 до большего значения"

Я за этот вариант, конечно.  с Warfly никуда перескакивать нельзя, это бессмысленно. Нужно его тайтлы масштабировать (растягивать) самим. Пользователи будут счастливы.
